sponsors/admin.py

```diff
@@ -196,6 +197,36 @@ def get_prepopulated_fields(self, request, obj=None):
             return {'slug': ['name']}
         return {}
 
+    def get_benefit_split(self, obj: SponsorshipPackage) -> str:
```

Comment:
I'd reccommend hard-coding here since the admin pallete is not going to consistently have all values populated.

With the prod  theme for instance this looks like:

<img width="254" alt="Screenshot 2024-09-11 at 4 22 30 PM" src="https://github.com/user-attachments/assets/91480775-4a5c-4b91-8922-42835e220901">


Something like

```python
    def get_benefit_split(self, obj: SponsorshipPackage) -> str:
        colors = [
            "#ffde57",  # Python Gold
            "#4584b6",  # Python Blue
            "#646464",  # Python Grey
        ]
        split = obj.get_default_revenue_split()
        # rotate colors through our available palette
        if len(split) > len(colors):
            colors = colors * (1 + (len(split) // len(colors)))
        # build some span elements to show the percentages and have the program name in the title (to show on hover)
        widths, spans = [], []
        for i, (name, pct) in enumerate(split):
            pct_str = f"{pct:.0f}%"
            widths.append(pct_str)
            spans.append(f"<span title='{name}' style='background-color:{colors[i]}'>{pct_str}</span>")
        # define a style that will show our span elements like a single horizontal stacked bar chart
        style = f'color:#fff;text-align:center;cursor:pointer;display:grid;grid-template-columns:{" ".join(widths)}'
        # wrap it all up and put a bow on it
        html = f"<div style='{style}'>{''.join(spans)}</div>"
        return mark_safe(html)
```

is probably fine and gives a lil pythonic flair!

<img width="209" alt="Screenshot 2024-09-11 at 4 21 14 PM" src="https://github.com/user-attachments/assets/00b96de0-db8f-4181-b08c-4c7084032342">